libnmstate/nm/ipv6.py

```diff
@@ -46,15 +52,22 @@ def get_info(active_connection):
 
 def create_setting(config):
     setting_ip = nmclient.NM.SettingIP6Config.new()
+    setting_ip.props.method = nmclient.NM.SETTING_IP6_CONFIG_METHOD_IGNORE
```

Comment:
Why is this needed here?

Reply:
Without it, the unit test will fail, I haven't look into the detail yet. Will do.

Reply:
In unit test, the `settting_ip.props.address` will be set:
`<MagicMock name='NM.SettingIP6Config.new().props.addresses' id='139895920922704'>`
which is not empty list.
I have applied a workround in unit test for this.